java/newton-core/src/main/java/io/muoncore/newton/saga/InMemorySagaRepository.java

```diff
@@ -73,7 +73,7 @@ private void registerEventExpectation(SagaInterest sagaInterest) {
   @Override
   public List<SagaInterest> getSagasInterestedIn(Class<? extends NewtonEvent> eventClass) {
     String eventName = eventClass.getName();
-    return sagaInterestStore.get(eventName);
+    return sagaInterestStore.getOrDefault(eventName, Collections.emptyList());
```

Comment:
That is so totally what I was looking for!

Reply:
:-)